Rakefile

```diff
@@ -92,7 +92,10 @@ end
 
 desc 'Bootstrap Carthage.'
 task :bootstrap do
-  safe_sh "carthage bootstrap"
```

Comment:
I think this can be done in one line, `carthage bootstrap --platform ios tvOS macOS`